msdk-id-sirius/src/main/java/SiriusIdentificationMethod.java

```diff
@@ -0,0 +1,95 @@
+import de.unijena.bioinf.ChemistryBase.chem.PrecursorIonType;
+import de.unijena.bioinf.ChemistryBase.ms.Ms2Experiment;
+import de.unijena.bioinf.ChemistryBase.ms.Peak;
+import de.unijena.bioinf.ChemistryBase.ms.Spectrum;
+import de.unijena.bioinf.sirius.IdentificationResult;
+import de.unijena.bioinf.sirius.IsotopePatternHandling;
+import de.unijena.bioinf.sirius.Sirius;
+import io.github.msdk.MSDKException;
+import io.github.msdk.MSDKMethod;
+import io.github.msdk.MSDKRuntimeException;
+import io.github.msdk.datamodel.IonAnnotation;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import javax.annotation.Nullable;
+
+/**
+ * Created by evger on 14-May-18.
+ */
+
+public class SiriusIdentificationMethod implements MSDKMethod<IonAnnotation> {
+  Sirius sirius;
+
+  SiriusIdentificationMethod() {
+    sirius = new Sirius();
+  }
+
+  /* This function is left here for non-msp files */
+  public Pair<double[], double[]> readCustomMsFile(String path, String delimeter) throws IOException, MSDKRuntimeException {
+    Scanner sc = new Scanner(new File(path));
+    ArrayList<String> strings = new ArrayList<>();
+    while (sc.hasNext()) {
+      strings.add(sc.nextLine());
+    }
+    sc.close();
+
+    double mz[] = new double[strings.size()];
+    double intensive[] = new double[strings.size()];
+
+    int index = 0;
+    for (String s : strings) {
+      String[] splitted = s.split(delimeter);
+      if (splitted.length == 2) {
+        mz[index] = Double.parseDouble(splitted[0]);
+        intensive[index++] = Double.parseDouble(splitted[1]);
+      } else throw new MSDKRuntimeException("Incorrect spectrum structure");
+    }
+
+    return new Pair<>(mz, intensive);
+  }
+
+  public List<IdentificationResult> identifyMs2Spectrum(
+      @Nullable Pair<double[], double[]> ms1pair, Pair<double[], double[]> ms2pair,
```

Comment:
Remove the Pair class, the method should just accept MsSpectrum objects as parameters